workspace/utilities/lib/menu-admin.xsl

```diff
@@ -7,80 +7,88 @@
+					<div class="menu-title">
+						<b>Admin</b>
+					</div>
+					<div class="menu-items-ctn">
+						<div class="menu-items-wrapper">
+							<div class="menu-item menu-item-symphony">
+								<a data-action="full" href="/symphony/">Symphony</a>
+							</div>
 
-				<span><xsl:text> - </xsl:text></span>
+							<xsl:if test="/data/events/login-info/@user-type = 'developer'">
+								<div class="menu-item menu-item-debug">
+									<a data-action="full">
+										<xsl:attribute name="href">
+											<xsl:text>?debug</xsl:text>
+											<xsl:if test="string-length(/data/params/current-query-string) != 0">
+												<xsl:text>&amp;</xsl:text>
+												<xsl:value-of select="/data/params/current-query-string" />
+											</xsl:if>
+										</xsl:attribute>
+										<xsl:text>Debug</xsl:text>
+									</a>
+								</div>
+							</xsl:if>
 
-				<a data-action="full" href="/symphony/">Symphony</a>
+							<xsl:if test="count(/data/params/use-dev) != 0 and $dev = true()">
+								<xsl:choose>
+									<xsl:when test="$debug = true()">
+										<xsl:if test="/data/events/login-info/@user-type = 'developer'">
+											<div class="menu-item menu-item-use-build">
+												<a data-action="full" href="?use-dev=no">
+													<p><xsl:text>Use Build</xsl:text></p>
+												</a>
+											</div>
+										</xsl:if>
+									</xsl:when>
+									<xsl:otherwise>
+										<div class="menu-item menu-item-build-info">
+											<strong>
+												<xsl:choose>
+													<xsl:when test="string-length(/data/build/last) != 0">
+														<xsl:text>Build </xsl:text>
+														<xsl:value-of select="/data/build/last" />
+													</xsl:when>
+													<xsl:when test="$dev = true()">
+														<xsl:text>No build infos found.</xsl:text>
+													</xsl:when>
+												</xsl:choose>
+											</strong>
+										</div>
 
-				<xsl:if test="/data/events/login-info/@user-type = 'developer'">
-					<a data-action="full">
-						<xsl:attribute name="href">
-							<xsl:text>?debug</xsl:text>
-							<xsl:if test="string-length(/data/params/current-query-string) != 0">
-								<xsl:text>&amp;</xsl:text>
-								<xsl:value-of select="/data/params/current-query-string" />
+										<xsl:if test="/data/events/login-info/@user-type = 'developer'">
+											<div class="menu-item menu-item-use-dev">
+												<a data-action="full" href="?use-dev">
+													<xsl:text>Use Dev</xsl:text>
+												</a>
+											</div>
+										</xsl:if>
+									</xsl:otherwise>
+								</xsl:choose>
 							</xsl:if>
-						</xsl:attribute>
-						<xsl:text>DEBUG</xsl:text>
-					</a>
-				</xsl:if>
 
-				<xsl:if test="count(/data/params/use-dev) != 0 and $dev = true()">
-					<xsl:choose>
-						<xsl:when test="$debug = true()">
-							<xsl:if test="/data/events/login-info/@user-type = 'developer'">
-								<span class="debug">
-									<a data-action="full" href="?use-dev=no">
```

Comment:
Can we keep this feature please ?

Reply:
Still there (new file, line 42), juste terribly unclear in the diff because of the changes in the structure.

Reply:
<3